src/System.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -19,9 +19,9 @@
 namespace System.IdentityModel.Tokens
 {
     /// <summary>
-    /// Error codes and messages
+    /// Log messages and codes
```

Comment:
One way to think about splitting this up is to have each assembly own their logging messages. So S.IM.Tokens has S.IM.Tokens/LogMessages.cs, etc.